pkgs/applications/misc/kile-wl/default.nix

```diff
@@ -1,20 +1,31 @@
-{ lib, fetchFromGitLab, rustPlatform }:
+{ lib, fetchFromGitLab, unstableGitUpdater, rustPlatform, scdoc }:
 
 rustPlatform.buildRustPackage rec {
   pname = "kile-wl";
-  version = "unstable-2021-04-22";
+  version = "unstable-2021-06-01";
 
   src = fetchFromGitLab {
     owner = "snakedye";
     repo = "kile";
-    rev = "b97b9f1e5b33862b33918efaf23fd1c0c5d7058a";
-    sha256 = "sha256-97qJd3o8nJt8IX5tyGWtAmJsIv5Gcw1xoBFwxAqk7I8=";
+    rev = "5d8dcbdb908e1f8bdc37a731c6e1c5d8c2ace3ba";
+    sha256 = "sha256-f3S91Ui6SKcbkmlHqqjlx4M/Te+2no75W3eAVKd769s=";
+  };
+
+  passthru.updateScript = unstableGitUpdater {
+    url = "https://gitlab.com/snakedye/kile.git";
```

Comment:
That won't update the cargo.lock, right?

Reply:
Cargo.lock is ignored by upstream

Reply:
This should be fixed now that upstream is tracking Cargo.lock.

Reply:
I'm not sure if this updates the `cargoSha256`, does it?

Reply:
True, is there a way to bypass that or update it in the script?

Reply:
nix-update can do that.